### ISSUE TITLE
Why is light mode pay walled?

### ISSUE THREAD

**Issue by meatsnails** [kawaiiDango/pScrobbler#922]
- My question is not listed in the FAQ at [faq.md](https://github.com/kawaiiDango/pano-scrobbler/blob/main/faq.md) 
 
This is just pay walling a basic accessibility feature it's way harder to read white in black text for me so black on white text being behind a "donation" is like really I mean it's a useful app but it would be a lot nicer if I could read things without having to strain my eyes please for the love of God why

**Comment by meatsnails** [comment]
why is the contrast settings also paywalled like what the fuck man

**Comment by meatsnails** [comment]
are you gonna respond
